src/main/java/ecdar/presentations/ComponentOperatorPresentation.java

```diff
@@ -146,13 +147,13 @@ public Color.Intensity getColorIntensity() {
      */
     @Override
     public ItemDragHelper.DragBounds getDragBounds() {
-        final ObservableDoubleValue minX = new SimpleDoubleProperty(Grid.GRID_SIZE);
+        final ObservableDoubleValue minX = new SimpleDoubleProperty(Ecdar.CANVAS_PADDING);
         final ObservableDoubleValue maxX = controller.getSystem().getBox().getWidthProperty()
-                .subtract(Grid.GRID_SIZE)
+                .subtract(Ecdar.CANVAS_PADDING)
                 .subtract(controller.getOperator().getBox().getWidth());
-        final ObservableDoubleValue minY = new SimpleDoubleProperty(Grid.TOOL_BAR_HEIGHT + Grid.GRID_SIZE * 3);
+        final ObservableDoubleValue minY = new SimpleDoubleProperty(Ecdar.CANVAS_PADDING * 5);
         final ObservableDoubleValue maxY = controller.getSystem().getBox().getHeightProperty()
-                .subtract(Grid.GRID_SIZE)
+                .subtract(Ecdar.CANVAS_PADDING * 5)
```

Comment:
Same as before: This is 5x the value it was previously, is this an error?

Reply:
Changed to `Ecdar.CANVAS_PADDING`, but has no effect